src/themes/palette.ts

```diff
@@ -27,6 +27,7 @@ export const palette = {
   greyLighter: '#8E90A6', //for placeholders and secondary button, border
   grayTransparent: 'rgba(98, 108, 116, 0.4)', //model backdrop
   shadow: ' 0px 16px 32px rgba(221, 230, 237, 0.4)',
+  linearGradient: 'linear-gradient(89.58deg, #44BCF0 -19.85%, #818CF8 54.07%, #A099FF 120.75%)',
```

Comment:
Name this gradient color accordingly. Maybe `blueIndigoGradient`

Reply:
yes, renamed in the latest commit.